Cargo.toml

```diff
@@ -18,3 +18,6 @@ default = ["std"]
 [package.metadata.release]
 no-dev-version = true
 tag-name = "{{version}}"
+
+[dependencies]
+serde = { version = "1.0", features = ["derive"], optional = true }
```

Comment:
Can you add a trailing new line, some linux tools (ie `wc`) expect the last line to end in a new line.

Reply:
Thanks, I've added a new line